lib/archival_storage_ingest/ingest_utils/ingest_utils.rb

```diff
@@ -129,6 +129,14 @@ def process(dir)
     end
   end
 
+  def self.truthy?(param:, default: false)
```

Comment:
I'm a bit hesitant about this function name because this is a different usage of 'truthy' from the Ruby standard (i.e., truthy == !(false || nil)). As an example, `truthy?(param: 33) == false`, but 33 is a truthy value.

Reply:
Makes sense.
Do you have any suggestion for the name?
Would parse_boolean_param work?

Reply:
Maybe `boolean_from_param`?

Reply:
Sounds good, I will update them.
Thanks.

Reply:
@Baroquem 
Could you check the updates I made updates as suggested?
P.S. I am not sure what that pull-request failure is, maybe we can ignore it?
Thanks.